envVariables.js

```diff
@@ -127,6 +127,11 @@ let enviromentVariables = {
 		message: 'Entity management service name required',
 		optional: false,
 	},
+	SUBMISSION_LEVEL: {
```

Comment:
@Prajwal17Tunerlabs Add this to env sample as well

Reply:
it is already added to envSample file @VISHNUDAS-tunerlabs